src/modules.js/saveDb.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable  */
```

Comment:
- [ ] kindly remove all of the `/* eslint-disable */` commands from all of your files and try to fix the linter errors 🙏 that way you will keep your code clean and readable

Reply:
done